timelock/program/src/processor/process_create_empty_timelock_config.rs

```diff
@@ -0,0 +1,61 @@
+//! Program state processor
+
+use crate::{
+    state::timelock_config::TimelockConfig,
+    state::timelock_program::TimelockProgram,
+    utils::{assert_initialized, assert_token_program_is_correct, create_account_raw},
+};
+use solana_program::{
+    account_info::{next_account_info, AccountInfo},
+    entrypoint::ProgramResult,
+    pubkey::Pubkey,
+};
+
+/// Create empty timelock config
+pub fn process_create_empty_timelock_config(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+) -> ProgramResult {
+    let account_info_iter = &mut accounts.iter();
+    let timelock_config_account_info = next_account_info(account_info_iter)?;
+    let program_to_tie_account_info = next_account_info(account_info_iter)?;
+    let governance_mint_account_info = next_account_info(account_info_iter)?;
+    let council_mint_account_info = next_account_info(account_info_iter)?;
```

Comment:
You may want some checks that these are actually mints owned by the token program

Reply:
good idea!